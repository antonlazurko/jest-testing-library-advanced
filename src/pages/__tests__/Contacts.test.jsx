import { screen, render } from "@testing-library/react";
import { Contact } from "../Contact";

describe("Contact", () => {
    test("should render", () => {
        render(<Contact />);
        expect(screen.getByRole('heading')).toMatchSnapshot();
    });
});