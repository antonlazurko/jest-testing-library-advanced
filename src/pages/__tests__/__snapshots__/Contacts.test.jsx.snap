// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render 1`] = `
<h1>
  Hello, from Contact page
</h1>
`;
